packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -64,35 +107,21 @@ const AccountRecoveryResetPassword = ({
       },
     });
 
-  const alertSuccessAndNavigate = useCallback(() => {
-    const successCompletePwdReset = ftlMsgResolver.getMsg(
-      `${viewName}-success-alert`,
-      'Password set'
+  // Make some presentation decisions based on initial state
+  if (state.contextError) {
+    alertInvalidContext(state.contextError);
+  } else if (!state.supportsRecovery) {
+    const msg = ftlMsgResolver.getMsg(
+      'redirecting',
```

Comment:
Is there an ftl entry for this message? I did a quick search but couldn't find any.

Reply:
No not yet, not sure if we are keeping it... But good call out. If we do keep it. We'll have to add something or pick a different message.

Reply:
Message added and id updated.